tests/test_pibi.py

```diff
@@ -1,7 +1,14 @@
-from src.ex1 import *
```

Comment:
Very bad practice to use star imports: it's wasteful, prevents tools from analyzing the code and it might cause namespace clashes that are difficult to debug